package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "vscode-env0",
   "publisher": "env0",
+  "icon": "resources/env0-logo.png",
```

Comment:
was there no icon up until now? I remember there was

Reply:
nope, I added it

Reply:
it's the icon for the sidebar, right?
I remember that there was one when I QA-ed a pr before

Reply:
it's for the marketplace @Wassap124